tests/unit/ext/auth/database.py

```diff
@@ -224,23 +238,25 @@ class TestUserModel(ModelTest):
     """
 
     def sample_obj(self):
-        return User('charlie', 'secret')
+        return dbauth().User('charlie', 'secret')
 
 
 admin_calls = [
-    (user_create, ['charlie', '1337']),
-    (user_create, ['charlie', '1337', False]),
-    (user_create, ['charlie', '1337', True]),
-    (user_delete, ['bob']),
-    (user_add_project, ['bob', 'runway']),
-    (user_remove_project, ['alice', 'runway']),
+    ('user_create', ['charlie', '1337']),
+    ('user_create', ['charlie', '1337', False]),
+    ('user_create', ['charlie', '1337', True]),
+    ('user_delete', ['bob']),
+    ('user_add_project', ['bob', 'runway']),
+    ('user_remove_project', ['alice', 'runway']),
 ]
```

Comment:
Because the function isn't in scope at the top level. It can't be without doing a global import, and that's what this patch is removing due to issues of contamination. Instead, the tests now use `getattr` to get the actual function.